samples/AzureStorage/AzureStorageLibrary/Blob/AzureBlobStorage.Codeunit.al

```diff
@@ -416,13 +442,16 @@ codeunit 50171 AzureBlobStorage
         end;
 
         // Add Authorization header
+        resourcePath := TypeHelper.UriEscapeDataString(resourcePath).Replace('%2F', '/');
```

Comment:
Why this? Do you have an example where it fails?

Reply:
This is needed to support a prefix with folder and subfolder support: ListBlobs -Prefix container/MainFolder/SubFolder/Document-